Interfaces/API/Dirac.py

```diff
@@ -65,6 +66,13 @@
 
 COMPONENT_NAME = 'DiracAPI'
 
+try:
```

Comment:
Why not `if six.PY2` here?

Reply:
Its hard to explain but feels more natural to use `try...except` here. I want to check if `file` is defined rather than use my knowledge about the Python versions involved.

Reply:
There are other cases where try..except are used, but then six wasn't imported, so I felt I had to ask.
It isn't a problem, as long as no one does something like `file = 'myfile.txt'` somewhere above (unlikely, but 🤷).